atc/lidar/scanner.go

```diff
@@ -18,23 +18,37 @@ func NewScanner(
 	secrets creds.Secrets,
 	defaultCheckTimeout time.Duration,
 	defaultCheckInterval time.Duration,
+	defaultWithWebhookCheckInterval time.Duration,
 ) *scanner {
+	// In case that a user configures resource-checking-interval, but forgets to
+	// configure resource-with-webhook-checking-interval, keep both checking-
+	// intervals consistent. Even if both intervals are configured, there is no
+	// reason webhooked resources take shorter checking interval than normal
+	// resources.
+	if defaultWithWebhookCheckInterval < defaultCheckInterval {
```

Comment:
Sorry one last thing before merging. Can you move this into the atccmd/command.go. It's configuration related, and we try to deal with all configuration at the top level.